pip/PhylumExt.toml

```diff
@@ -0,0 +1,10 @@
+name = "pip"
+description = "pip package manager hooks"
+entry_point = "main.ts"
+
+[permissions]
+run = ["/bin"]
```

Comment:
This is a placeholder. Going to figure out what to do about Python once I know about the macOS sandboxing requirements.